deliver/lib/deliver/upload_screenshots.rb

```diff
@@ -32,7 +32,11 @@ def upload(options, screenshots)
         lng_text = "language"
         lng_text += "s" if enabled_languages.count != 1
         UI.message("Activating #{lng_text} #{enabled_languages.join(', ')}...")
-        v.save!
+        begin
+          v.save!
+        rescue ITunesConnectError
+          UI.user_error!("Unable to activate languages: #{langs}. Please verify that your language codes are available in iTunesConnect. See https://developer.apple.com/library/content/documentation/LanguagesUtilities/Conceptual/iTunesConnect_Guide/Chapters/AppStoreTerritories.html for more information.")
```

Comment:
Its definitely possible that this could be something else, but this is the most likely.